release-notes/major9/09.24.0/valtimo-frontend-libraries.md

```diff
@@ -19,6 +19,56 @@ The following features were added:
 
   The default language is used for every user that has not set a preferred language. If no language is set in the `environment.ts` the default is set to `Language.NL`.
 
+* **Valtimo list action callback**
```

Comment:
This shouldn't just be release notes at this point, but actual documentation.

Reply:
yeah I've been thinking of writing a page for the valtimo-list in its entirety tbh, think I'll do that and include include this example in the 'actions' section of that page